_sass/pages/_getting-help.scss

```diff
@@ -47,14 +47,15 @@
   .support-section,
   .service-section {
     padding-top: 40px;
-    padding-bottom: 40px;
 
     @media (max-width: $tablet-large) {
       padding-bottom: 24px;
     }
   }
 
   .support-section {
+    padding-bottom: 40px;
```

Comment:
Is this one connect to the `.recaptcha-note` bottom padding? If so, let's have a variable.

Reply:
It is an independent padding.